source/_docs/configuration/templating.markdown

```diff
@@ -169,16 +169,17 @@ The same thing can also be expressed as a filter:
 
 ### Time
 
-- `now()` will be rendered as the current time in your time zone.
-  - For specific values: `now().second`, `now().minute`, `now().hour`, `now().day`, `now().month`, `now().year`, `now().weekday()` and `now().isoweekday()`
-- `utcnow()` will be rendered as UTC time.
+Note: UNIX timestamp is the number of seconds that have elapsed since the Unix epoch and therefore can be substituted with `int` or `float` value when used as an argument of functions below.
```

Comment:
I don't think this addition makes it any clearer for users.
Please reformulate in a way, less technical users can understand it as well.

(And add a new line after it, before starting a list in Markdown).

Reply:
Actually, this addition is not aimed at less technical users so they can skip it without any harm.
I added it instead of writing something like this
```
- Filter `timestamp_local` converts an UNIX timestamp/float/int to a string
```
in at least 3 places.
What do you think?

Reply:
> Actually, this addition is not aimed at less technical users

Well, we should. Hence, my request.

Reply:
reworded it and added a link/example